src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,213 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size, screenSize } from './theme';
+import { P, ArticleH3, H3 } from './text';
+import Input from './input';
+import Button from './button';
+import FormSelect from './select';
+import Checkbox from '@leafygreen-ui/checkbox';
+import SectionHeader from './section-header';
+import SuccessIcon from './icons/success';
+
+const StyledButton = styled(Button)`
+    display: flex;
+    margin: ${size.large} auto;
+`;
+
+const StyledInput = styled(Input)`
+    height: 40px;
+    margin-bottom: ${size.large};
+    width: 335px;
+`;
+
+const StyledFormSelect = styled(FormSelect)`
+    color: ${colorMap.greyLightTwo};
+    height: 40px;
+    width: 335px;
```

Comment:
Same issue with mobile responsiveness with fixed px widths. Can we use something like Flexbox to help provide a more responsive experience?